src/core/src/SettingsWrapper.cpp

```diff
@@ -49,17 +47,59 @@ const std::vector<UserAccessToken> getAccessTokens() {
     return tokens;
 }
 
-void setAccessTokens(const std::vector<UserAccessToken>& accessTokens) {
+void setAccessToken(const UserAccessToken& userAccessToken) {
```

Comment:
See if parts of `setAccessToken` and `removeAccessToken` can be consolidated into a helper function

Reply:
Not going to worry about this right now